README.md

```diff
@@ -0,0 +1,64 @@
+# Portfolio-mobile-version
+Microverse project
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Veljko Panic**
+
+- GitHub: [@WillyStorm](https://github.com/WillyStorm)
+- Twitter: [@VeljaSTAR](https://twitter.com/VeljaSTAR)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/veljko-panic-437b12231/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the license is not working as expected, please have another look at [the Microverse template](https://github.com/microverseinc/readme-template) for an example of how it is expected to work.